gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,790 @@
+)
+logger = logging.getLogger("create_release_ht")
+logger.setLevel(logging.INFO)
+
+# Remove InbreedingCoeff from allele-specific fields because it is processed separately
+# from the other fields.
+AS_FIELDS = deepcopy(AS_FIELDS)
+AS_FIELDS.remove("InbreedingCoeff")
+# TODO: Julia needs to find "AS_SOR" and "SOR" if possible
+AS_FIELDS.remove("AS_SOR")
+SITE_FIELDS = deepcopy(SITE_FIELDS)
+SITE_FIELDS.remove("SOR")
+TABLES_FOR_RELEASE = [
+    "dbsnp",
+    "filters",
+    "freq",
+    "info",
+    "region_flags",
+    "in_silico",
+    "vep",
+    "joint_faf",
+]
+
+INSILICO_PREDICTORS = ["cadd", "revel", "spliceai", "pangolin", "phylop"]
+
+FINALIZED_SCHEMA = {
+    "globals": [
+        "freq_meta",
+        "freq_index_dict",
+        "freq_meta_sample_count",
+        "faf_meta",
+        "faf_index_dict",
+        "joint_freq_meta",
+        "joint_freq_index_dict",
+        "joint_freq_meta_sample_count",
+        "joint_faf_meta",
+        "joint_faf_index_dict",
+        "age_distribution",
+        "downsamplings",
+        "filtering_model",
+        "inbreeding_coeff_cutoff",
+        "tool_versions",
+        "vrs_versions",
+        "vep_globals",
+        # "frequency_README", # TODO: uncomment when exome release PR is in
+        "date",
+        "version",
+    ],
+    "rows": [
+        "freq",
+        "grpmax",
+        "faf",
+        "fafmax",
+        "joint_freq",
+        "joint_grpmax",
+        "joint_faf",
+        "joint_fafmax",
+        "a_index",
+        "was_split",
+        "rsid",
+        "filters",
+        "info",
+        "vep",
+        "vqsr_results",
+        "region_flags",
+        # "allele_info", # TODO: double check if this is needed
+        "histograms",
+        "in_silico_predictors",
+    ],
+}
+
+
+def drop_v3_subsets(freq_ht: hl.Table) -> hl.Table:
+    """
+    Drop the freq of all v3 subsets except HGDP + TGP from the freq Table.
+
+    :param freq_ht: v4.0 genomes freq Table
+    :return: v4.0 genomes freq Table with some v3 subsets dropped
+    """
+    SUBSETS_TO_DROP = {
+        "subset": [
+            "non_v2",
+            "non_topmed",
+            "non_cancer",
+            "controls_and_biobanks",
+            "non_neuro",
+        ]
+    }
+
+    freq_meta, array_exprs = filter_arrays_by_meta(
+        freq_ht.freq_meta,
+        {
+            "freq": freq_ht.freq,
+            "freq_meta_sample_count": freq_ht.index_globals().freq_meta_sample_count,
+        },
+        SUBSETS_TO_DROP,
+        keep=False,
+        combine_operator="or",
+    )
+
+    freq_ht = freq_ht.annotate(freq=array_exprs["freq"])
+    freq_ht = freq_ht.annotate_globals(
+        freq_meta=freq_meta,
+        freq_index_dict=make_freq_index_dict_from_meta(hl.literal(freq_meta)),
+        freq_meta_sample_count=array_exprs["freq_meta_sample_count"],
+    )
+
+    return freq_ht
+
+
+# Config is added as a function, so it is not evaluated until the function is called.
+def get_config(
```

Comment:
So far from what I see this is exactly the same as Mike's and can easily just have a `data_type: str = "exomes",` parameter added where all the places that have `data_type="genomes"` can just be changed to `data_type=data_type` and then the genomes and exomes release can use the same config